tests/integration/stack_component/dotnet/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Pulumi;
+using Pulumi.Serialization;
+
+class MyStack : Stack
+{
+    [Output("abc")]
```

Comment:
we should consider if name can be optional.  in which case we export using the property name.

Reply:
also for components?

Reply:
not certain what you mean.

Reply:
`OutputAttbiute` has no default constructor at the moment because names are required for normal resources' outputs. If we introduce the default constructor, that means that other resources may get `Output` without the explicit names. Should we take care of that?

Reply:
i think this is the right way to go (i.e. making name optional).  Our generators will do the right thing.  But this just makes things much cleaner and nicer for users in the common case (i.e. `[Output] ...`).